app/routes/users/components/RemovePicture.js

```diff
@@ -33,20 +34,23 @@ export default class RemovePicture extends Component<Props, State> {
 
   render() {
     return (
-      <Flex justifyContent="center">
-        <Button className={styles.saveButton} onClick={this.toggleSelected}>
-          {this.state.selected ? 'Angre' : 'Slett bilde'}
-        </Button>
+      <div className={styles.buttons}>
+        {this.state.selected ? (
+          <Button className={styles.saveButton} onClick={this.toggleSelected}>
```

Comment:
Here styles.savebutton is used on "Avbryt" to get 10px margin 👎🏻

Reply:
It was actually not used for anything, but should be removed anyway lol